package.json

```diff
@@ -11,14 +11,18 @@
     "postinstall": "pnpm compile",
     "compile": "pnpm -r -w=false run compile",
     "test:unit": "pnpm -r -F=./packages/** test",
-    "test:type": "pnpm -F=@sap-ai-sdk/type-tests test",
-    "test:e2e": "pnpm -F=@sap-ai-sdk/e2e-tests test",
+    "test:type": "pnpm type-tests test",
+    "test:e2e": "pnpm e2e-tests test",
+    "test:smoke": "pnpm smoke-test test",
     "lint": "pnpm -r run lint",
     "lint:fix": "pnpm -r run lint:fix",
     "generate": "pnpm -r run generate",
     "ai-core": "pnpm -F=@sap-ai-sdk/ai-core",
     "gen-ai-hub": "pnpm -F=@sap-ai-sdk/gen-ai-hub",
-    "core": "pnpm -F=@sap-ai-sdk/core"
+    "core": "pnpm -F=@sap-ai-sdk/core",
```

Comment:
can't find this option in the docs, what does `-F` do?

Reply:
It filters packages based on the provided selector. https://pnpm.io/filtering